install_bootstrap

```diff
@@ -218,7 +218,7 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
       dcc_dashboard_port=8080
 
       if [ -f boardwalk_launcher_config/boardwalk.config ] ; then
-        	source <(jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]" boardwalk_launcher_config/boardwalk.config)
+         source <(jq -r "to_entries|map(\"\(.key)=\(.value|tostring)\")|.[]" boardwalk_launcher_config/boardwalk.config)
```

Comment:
the didn't change. I think the editor is set to 4 spaces. in the deleted line it is indented by 8 spaces, but nowhere else in the code are indents 8 space long. do you still want to set it back to 8 spaces?

Reply:
What happened is the tab character got deleted.

Go ahead and leave it. It was the only tab character in the file, so you're not breaking a pattern.

The reason I suggested reverting it is because it makes things slightly more confusing. When I saw that line show up as changed in the GitHub GUI, I copied both lines to an editor to compare them, to see what had changed, to see what I needed to review. So in general, you don't want to make changes that aren't relevant to the issue, because it introduces noise.

But this change is small enough, and it makes the formatting a little better so let's go ahead as is.